athena/llm_core/llm_core/utils/append_format_instructions.py

```
@@ -0,0 +1,44 @@
+        )
+    elif isinstance(first_message, HumanMessagePromptTemplate):
+        new_first_message = HumanMessagePromptTemplate.from_template(
+            first_message.prompt.template + "<OutputFormat>\n\n{format_instructions}"  # type: ignore[attr-defined]
+        )
+    else:
+        raise ValueError(
+            f"Unsupported message type for appending format instructions: {type(first_message)}"
+        )
+
+    new_first_message.prompt.partial_variables = {"format_instructions": format_instructions}  # type: ignore[attr-defined]
+    new_first_message.prompt.input_variables.remove("format_instructions")  # type: ignore[attr-defined]
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)